hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFileSystem.java

```diff
@@ -63,7 +63,7 @@ public URI getKeyProviderUri() throws IOException {
     try {
       keyProvider = getKeyProvider();
     } catch (IOException ioe) {
-      LOG.error("Error retrieving KeyProvider.", ioe);
+      LOG.debug("Error retrieving KeyProvider.", ioe);
```

Comment:
You can also update the text to say something like "this error is benign when no keyprovider is configured."